code-challenges.js

```diff
@@ -13,21 +26,53 @@ var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
 var fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
 // Expected output: [-823, 7, 23]
 
+let numsOnly = (value) => {
+    if(typeof(value)=== 'number' && value % 2 !== 0){
+        return value
+    }
+}
+// .filter() here that will create an array with only numbers and then put them in order from least to greatest with .sort()
+let oddNumsOnly = (array) => {
+    return array.filter(numsOnly).sort(function(a, b){return a-b})
+}
```

Comment:
Totally good idea here. But I'm curious why you broke this into two different functions? This could be inside the scope of the numsOnly and all packaged up as one beautiful function! :) 

Super good logic here though!